doc/doxygen/references.bib

```diff
@@ -1089,3 +1089,22 @@ @article{Geevers_2018
    month={Jan},
    pages={A2830–A2857}
 }
+
+@article{witherden2015identification,
+  title={On the identification of symmetric quadrature rules for finite element methods},
+  author={Witherden, Freddie D and Vincent, Peter E},
+  journal={Computers \& Mathematics with Applications},
+  volume={69},
+  number={10},
+  pages={1232--1241},
+  year={2015},
+  publisher={Elsevier}
+}
+
+@online{quadpy,
```

Comment:
Is that an actual category that doxygen understands?

Reply:
I think that quadpy has a DOI. You can always convert DOIs to bibtext format at https://www.doi2bib.org

Reply:
It worked with my copy of doxygen.